AppAuth-JS/built/redirect_based_handler.d.ts

```diff
@@ -0,0 +1,22 @@
+import { AuthorizationRequest } from './authorization_request';
```

Comment:
This is generated source file. All the files in built/ folder are generated JS file for the Google's lib fork